starters/features/drizzle/package.json

```diff
@@ -22,7 +22,11 @@
         " Check out the Drizzle docs for more info:",
         "   - https://orm.drizzle.team/docs/overview"
       ]
-    }
+    },
+    "alwaysInRoot": [
+      "drizzle",
+      "drizzle.config.ts"
+    ]
```

Comment:
really? We have drizzle in Insights and it's in the insights package dir

Reply:
good catch, thanks
fixing